src/js/components/tasks/detail.tsx

```diff
@@ -115,6 +184,23 @@ const TaskDetail = (props: RouteComponentProps) => {
     </PageHeaderControl>
   );
 
+  let buttonText: string | React.ReactNode = i18n.t('Capture Task Changes');
+  if (currentlyCommitting) {
+    buttonText = (
+      <LabelWithSpinner
+        label={i18n.t('Capturing Selected Changes…')}
+        variant="inverse"
+      />
+    );
+  } else if (fetchingChanges || currentlyFetching) {
+    buttonText = (
+      <LabelWithSpinner
+        label={i18n.t('Checking for Uncaptured Changes…')}
```

Comment:
Do we want these things to be title-case?

Reply:
I'd leave that up to @stacyk and @SondraE, but it does match the CTA buttons elsewhere through the app.